Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10
+
+# Install package dependencies
+RUN DEBIAN_FRONTEND='noninteractive' apt-get -y update && \
+    DEBIAN_FRONTEND='noninteractive' apt-get -y upgrade
+RUN DEBIAN_FRONTEND='noninteractive' apt-get install -y --no-install-recommends netcat arp-scan
+
+RUN mkdir -p /app && \
+    mkdir -p /tmp
+
+WORKDIR /app
+
+COPY package.json ./
+
+RUN npm install
```

Comment:
This isn't getting cached, which causes the build to take a long time. `Dockerfile` should be reworked so that this operation is only executed if `package.json` changes.

A good article on how to do this: http://bitjudo.com/blog/2014/03/13/building-efficient-dockerfiles-node-dot-js/